### ISSUE TITLE
TextView bug (Integer Decimal Number)

### ISSUE THREAD

**Issue by ckarmy** [fuse-open/fuselibs#1399]
When I focus a textview Integer, Decimal or Number when I release the input the text dissapear. I upload a video as example:
https://user-images.githubusercontent.com/10090783/108730334-32f56d80-750a-11eb-9b4c-4ee155139003.mp4

**Comment by ichan-mb** [comment]
Could you post the code to reproduce the error. I couldn't replicate the error on my device (Xiaomi Mi 10, Android 10). Here is my code:

```
<TextView Height="100" TextWrapping="Wrap" Padding="4" Margin="4" TextColor="#000" CaretColor="#000". InputHint="Decimal">
	<Rectangle Layer="Background" CornerRadius="4">
		<Stroke Color="#000" />
	</Rectangle>
</TextView>
```

**Comment by ckarmy** [comment]
@ichan-mb  your code it works for me too. But it happens if you add `TextAlignment="Center"` to the `TextView` like:

```
<TextView Height="100" TextAlignment="Center" TextWrapping="Wrap" Padding="4" Margin="4" TextColor="#000" CaretColor="#000" InputHint="Decimal">
	<Rectangle Layer="Background" CornerRadius="4">
	        <Stroke Color="#000" />
        </Rectangle>
</TextView>
```

**Comment by Anagrr** [comment]
What if you decrease the font size for 1 or 2 units? )) As I see from the video, when input is in focus the carret is taller than font almost on 5 units. So while focused, input increases its size a little bit I assume. It's just my opinion which might be wrong

**Comment by ichan-mb** [comment]
@ckarmy yes, I can confirm the issue and found the workaround how to fix it. I hope I can make the PR soon.

**Comment by ichan-mb** [comment]
This issue has been fixed on the master branch and will be released on latest v2.0 beta. I'm closing the issue now. If problem still persists feel free to comment to this issue. Thanks!
